cookbooks/bcpc-hadoop/attributes/hbase.rb

```diff
@@ -71,6 +73,7 @@
 default[:bcpc][:hadoop][:hbase][:site_xml].tap do |site_xml|
   site_xml['hbase.rootdir'] = "#{node['bcpc']['hadoop']['hbase']['root_dir']}"
   site_xml['hbase.bulkload.staging.dir'] = "#{node['bcpc']['hadoop']['hbase']['bulkload_staging_dir']}"
+  site_xml['hbase.fs.tmp.dir'] = "#{node['bcpc']['hadoop']['hbase']['fs_tmp_dir']}"
```

Comment:
Please don't create vanity names for site.xml variables; simply set the site.xml variable.

Reply:
@cbaenziger  the change was done this way to be consistent with what seems to be a standard followed in the code historically. We can make a break with this change.

Reply:
Hi Biju, yes, please continue to move away from these vanity names. We are working to unify on the HBase community names which allows us to use the variables directly in the `site_xml` hash. Thanks!